Procfile

```diff
@@ -1,2 +1,3 @@
-server: bundle exec bin/rails server -e production
-anycable: bundle exec anycable --server-command "anycable-go --host=localhost --port 3334"
+server:      RAILS_LOG_TO_STDOUT=true bundle exec bin/rails server -e production -b '0.0.0.0' -p 80
+anycable-go: RAILS_LOG_TO_STDOUT=true bundle exec bin/rails runner -e production "exec(\"anycable-go --host=0.0.0.0 --port 3334 --redis_url='#{Exercism.config.anycable_endpoint}'\")"
```

Comment:
Where is `Exercism.config.anycable_endpoint` defined?

Reply:
My guess is that command is hackingly injecting a ruby string. So it would be defined in one of the initializers.

Reply:
So this is the `ExercismConfig` gem. It preloads `Exercism.config` with everything you need in any environment, loaded from either local/docker-composed or production dynamodb. Coming in a funky PR shortly.

Reply:
Okay cool!